src/env.ts

```diff
@@ -0,0 +1,3 @@
+export default {
+  sealCredAddress: 'https://sealcred.xyz/',
```

Comment:
We can use real `dotenv`, but I think it's not necessary, because this works for production and dev can change it anytime